packages/driver/src/dom/elements.js

```diff
@@ -387,6 +390,10 @@ const isFocusable = ($el) => {
   })
 }
 
+const isFocusableAndNotHidden = (el) => {
```

Comment:
We are overloading the term `hidden` because hidden has special semantic meaning to Cypress and our own algorithm.

This code is attempting to match the **w3c definition** of hidden, which is completely unclear and opaque. 

This code not only needs comments indicating this distinction, but it could also use either the w3c algorithm dropped in as a comment - or instead, a link to the w3c definition. Ideally, both

Reply:
ah, the correct term is `inert` https://html.spec.whatwg.org/multipage/interaction.html#focusable-area